rca/account_management/templates/account_management/create.html

```diff
@@ -37,14 +37,3 @@
             </section>
     </form>
 {% endblock %}
-
-{% block extra_css %}
-    {{ block.super }}
-    {% include "wagtailadmin/pages/_editor_css.html" %}
-    {{ form.media.css }}
-{% endblock %}
-{% block extra_js %}
-    {{ block.super }}
-    {% include "wagtailadmin/pages/_editor_js.html" %}
-    {{ form.media.js }}
-{% endblock %}
```

Comment:
Removed these blocks since these don't seem to be supported anymore.

Reply:
It looks like it's only this thats now removed.
```
{% block extra_css %}
    {{ block.super }}
    {% include "wagtailadmin/pages/_editor_css.html" %}
    {{ form.media.css }}
{% endblock %}
```
the `_editor_js.html` still exists.

Reply:
Hello, I have reverted this commit: [bb9c380338387186e2d930cdea41eed32e2540b3](https://github.com/torchbox/rca-wagtail-2019/pull/904/commits/bb9c380338387186e2d930cdea41eed32e2540b3),
and only deleted the extra_css block for this template.